### PR TITLE
Todo-12-FE: As a user, I want to sort todo items in my list

### PR DESCRIPTION
**Pull request by itc02**
image
image
image

### CODE REVIEW

**Comment by Deathlive** [comment]
Mihat harch
Orinak xechnq ogtagortum [https://material-ui.com/components/tables/#sorting-amp-selecting](Sorting & Selecting) ?

File: src/components/mainTable/MainTable.tsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ export const MainTable:React.FC = () => {
   const [ allTodosCount, setAllTodosCount ] = useState<number>(0);
   const [ chosenTodos, setChosenTodos ] = useState<number[]>([]);
 
+  const [ sortingCriteria, setSortingCriteria ] = useState<string>('');
+  const [ order, setOrder ] = useState<string>('');
```
Comment (Deathlive) [inline src/components/mainTable/MainTable.tsx:35]: Guche by default ini `ASC` kam `DESC` ?

**Review by Demigoddd — APPROVED** [review]

**Comment by itc02** [comment]
@Deathlive Prosto react-material-in sorting-y hnaravora en depqum, ete sax todo-nen miangamic serveran kyamn. Isk im depqum en todo-nenn kyam, voronq hamapatasxanumn tvyal per-in ev page-in.

**Review by Demigoddd — COMMENTED** [review]
test review

**Pull request merged**